### PR TITLE
Added endpoint to get messages for a movement with LRN

### PR DESCRIPTION
**Pull request by solomonamadi**

### CODE REVIEW

**Review by ChrisCookOC — COMMENTED** [review]
Can you update the API yaml to say LRN instead of movementId?

File: app/uk/gov/hmrc/excisemovementcontrolsystemapi/repository/MovementMessageRepository.scala [hunk 1]
```diff
@@ -64,4 +64,17 @@ class MovementMessageRepository @Inject()
       .toFuture()
       .map(_ => true)
   }
+
+  def getMovementMessagesForERN(ern: String): Future[Seq[MovementMessage]] = {
```
Comment (mauro-fanton) [inline app/uk/gov/hmrc/excisemovementcontrolsystemapi/repository/MovementMessageRepository.scala:68]: please, add test for this. Thanks

File: app/uk/gov/hmrc/excisemovementcontrolsystemapi/repository/MovementMessageRepository.scala [hunk 2]
```diff
@@ -64,4 +64,17 @@ class MovementMessageRepository @Inject()
...
+    collection.find(or(equal("consignorId", ern), equal("consigneeId", ern))).toFuture()
+  }
+
+  def getMovementMessagesForERNList(erns: List[String]): Future[Seq[MovementMessage]] = {
```
Comment (mauro-fanton) [inline app/uk/gov/hmrc/excisemovementcontrolsystemapi/repository/MovementMessageRepository.scala:72]: as above.

File: app/uk/gov/hmrc/excisemovementcontrolsystemapi/repository/MovementMessageRepository.scala [hunk 3]
```diff
@@ -64,4 +64,17 @@ class MovementMessageRepository @Inject()
...
+    collection.find(or(in("consignorId", erns: _*), in("consigneeId", erns: _*))).toFuture()
+  }
+
+  def getMovementMessagesByLRNAndERNIn(lrn: String, erns: List[String]): Future[Seq[MovementMessage]] = {
```
Comment (mauro-fanton) [inline app/uk/gov/hmrc/excisemovementcontrolsystemapi/repository/MovementMessageRepository.scala:76]: as above

**Review by ChrisCookOC — APPROVED** [review]

**Pull request merged**